app/views/partials/personal_key/_key.html.erb

```diff
@@ -7,10 +7,17 @@
     </div>
   </div>
   <p class="margin-bottom-105">
-    <%= t(
-          'users.personal_key.generated_on_html',
-          date: content_tag(:strong, I18n.l(Time.zone.today, format: '%B %d, %Y')),
-        ) %>
+    <% if local_assigns[:personal_key_generated_at].present? %>
+      <%= t(
+            'users.personal_key.generated_on_html',
+            date: content_tag(:strong, render(TimeComponent.new(time: personal_key_generated_at))),
+          ) %>
+    <% else %>
+      <%= t(
+            'users.personal_key.generated_on_html',
+            date: content_tag(:strong, render(TimeComponent.new(time: Time.zone.today))),
+          ) %>
+    <% end %>
```

Comment:
what are the cases when this is nil? is it only when it's used as an example?

If so, would we want to render some sort of example time, instead of "now" like "[TIMESTAMP]" as a clear placeholder?

Reply:
The fallback can likely be removed. I believe the references to the partials `shared/personal_key` and
`partials/personal_key/key` have been modified appropriately.

Previously Time.zone.today was often displayed, so I left the fallback there out of an abundance of caution.